language/functional_tests/tests/testsuite/transaction_fee_distribution/test_txn_fee_distribution_does_div.mvir

```diff
@@ -1,27 +1,52 @@
 // Number of validators evenly divides the transaction fees collected
 
+//! account: vivian, 1000000, 0, validator
+//! account: bobby, 1000000, 0, validator
```

Comment:
Come on man, let's see some creative "V" names :)

Reply:
Fixed! Had to google baby names though lol